pyproject.toml

```diff
@@ -10,7 +10,9 @@ requires-python = ">=3.11"
 dependencies = [
     "mcp[cli] (>=1.10,<1.11)",
     "httpx (>=0.28.1,<0.29.0)",
-    "pillow (>=11.2.1,<12.0.0)"
+    "pillow (>=11.2.1,<12.0.0)",
+    "caldav (>=1.3.6,<2.0.0)",
```

Comment:
What's the reason for the `caldav` dependency? It looks like this PR re-implements the bulk of what this library does